external-crates/move/crates/move-compiler/src/sui_mode/typing.rs

```diff
@@ -888,6 +888,23 @@ fn exp(context: &mut Context, e: &T::Exp) {
             if is_transfer_module && PRIVATE_TRANSFER_FUNCTIONS.contains(&name.value()) {
                 check_private_transfer(context, e.exp.loc, mcall)
             }
+            // TODO(tzakian): Temporary
```

Comment:
Is this part of the code well tested, if it's temporary? If not, is there a plan to address this in a follow-up PR?